dns-records/dhis2-beta-phac-aspc-gc-ca.yaml

```
@@ -0,0 +1,19 @@
+apiVersion: dns.cnrm.cloud.google.com/v1beta1
+kind: DNSRecordSet
+metadata:
+  name: dhis2-beta-phac-aspc-gc-ca
+  namespace: config-control
+  annotations:
+    projectName: "dhis2"
+    sourceCodeRepository: "https://github.com/PHACDataHub/dhis-2/"
+spec:
+  name: "dhis2.beta.phac-aspc.gc.ca."
+  type: "NS"
+  ttl: 300
+  managedZoneRef:
+    external: dhis2-beta-phac-aspc-gc-ca
```

the managed zone needs to point to the parent zone
```suggestion
beta-phac-aspc-gc-ca
```